code/datums/supplypacks.dm

```diff
@@ -1371,7 +1377,7 @@ var/list/all_supply_groups = list("Operations","Security","Hospitality","Enginee
 
 /datum/supply_packs/hydrotray
 	name = "Empty hydroponics tray"
-	cost = 30
+	cost = 10

```

Comment:
the cost on this was just stupidly high, call it a minor fix.

You need like eight of these things to setup a garden, theyre not very good individually